### PR TITLE
Jekyll site

### PR DESCRIPTION
**Pull request by benhylau**
Completes https://github.com/tomeshnet/p2p-internet-workshop/issues/63 by building and pushing a Jekyll site on tag.

### CODE REVIEW

**Comment by benhylau** [comment]
Changes archive directory structure to be more like https://github.com/tomeshnet/p2p-internet-workshop/issues/41 (per module) and ported theme from Mozilla's as suggested by @patcon https://github.com/tomeshnet/p2p-internet-workshop/issues/38#issuecomment-389034687.

**Comment by benhylau** [comment]
New site 🎆 https://tomeshnet.github.io/p2p-internet-workshop/

**Comment by benhylau** [comment]
If there are no objections I am just going to merge this in a couple days. There is some text change in the README, and a whole website modified from Moz's as base, changing structure of our course asset packaging. Quite a lot in this PR.

File: README.md [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
 
 ## Module Topics
 
-1. [What is mesh?](./module-1/README.md)
-1. [Setting up your first node](./module-2/README.md)
-1. [Building nodes (hardware)](./module-3/README.md)
-1. [Planning a network](./module-4/README.md)
-1. [Measuring networks](./module-5/README.md)
-1. [Living on a decentralized network](./module-6/README.md)
+1. [What is mesh?]({{ site.baseurl }}/articles/module-1/)
```
Comment (dcwalk) [inline README.md]: Does this work when just rendered at a md?

File: README.md [hunk 2]
```diff
@@ -35,15 +33,15 @@ Participants will be expected to bring their own laptop.
 
 ## Expectations
 
-- TPL will provide Raspberry Pi 2s and 3s
+- A set of hardware is available for each participant
```
Comment (dcwalk) [inline README.md]: This is oddly pluralized?

"Each participant will have a hardware kit" maybe?

File: README.md [hunk 3]
```diff
@@ -35,15 +33,15 @@ Participants will be expected to bring their own laptop.
...
 - Participants will bring their own laptop
-- Participants will receive primer information via a [welcome email](./general/welcome-email.txt), which includes [laptop setup instructions](./SETUP.md), a Linux shell primer, and a [pre-workshop survey](./general/pre-workshop-survey.md)
+- Participants will receive primer information via a [welcome email]({{ site.baseurl }}/articles/general/general-welcome-email.pdf), which includes [laptop setup instructions]({{ site.baseurl }}/articles/general/general-laptop-setup-instructions.pdf), a Linux shell primer, and a [pre-workshop survey]({{ site.baseurl }}/articles/general/general-pre-workshop-survey.pdf)
```
Comment (dcwalk) [inline README.md]: using site.baseurl breaks the readme, can we just use the actual url?

File: README.md [hunk 4]
```diff
@@ -54,38 +52,20 @@ Participants will be expected to bring their own laptop.
...
-Travis CI is configured to build, package, and publish a new release to GitHub Releases when you push a new tag, so to publish a new package, all you need is to find the current version with `git tag`, and if the next one should be `v0.4`, you do:
-
-    git tag v0.4
-    git push --tag
-
-When using the GitBook presentations at a workshop, you can either:
-
-- Load the appropriate module from GitHub Pages, such as [module-1-presentation](https://tomeshnet.github.io/p2p-internet-workshop/module-1-presentation/index.html)
-- Or serve the slides from your local machine by cloning this repository, navigate to the appropriate presentation directory of the module to run `gitbook serve`, then navigate to [localhost:4000](http://localhost:4000) from your browser to see the slides
+Class materials are written as [Markdown](https://en.wikipedia.org/wiki/Markdown) files and presentation slides are created as a Markdown-based [GitBook](https://www.gitbook.com). All generated assets are hosted on [GitHub Pages]({{ site.baseurl }}/) and packaged as a downloadable archive on [GitHub Releases](https://github.com/tomeshnet/p2p-internet-workshop/releases/latest).
```
Comment (dcwalk) [inline README.md]: Same comment about baseurl

File: README.md [hunk 5]
```diff
@@ -54,38 +52,20 @@ Participants will be expected to bring their own laptop.
...
 
-## Course Design
```
Comment (dcwalk) [inline README.md]: I think you should move these somewhere even if not in the README

File: README.md [hunk 6]
```diff
@@ -54,38 +52,20 @@ Participants will be expected to bring their own laptop.
...
-- Engage students and equip them with necessary knowledge to continue involvement in local mesh community efforts
-- Publish this as open-source syllabus that other communities can adopt to kick-start a mesh community
+[Travis CI](https://travis-ci.org/) is configured to build, package, and publish a new release to GitHub Pages and Releases whenever a new tag is pushed. So all you need to create a new release is to push a new tag with `git tag <version>` and `git push --tag`.
+  
+## Credits
 
-### Credits
+The first version of this syllabus is created by [Toronto Mesh](https://tomesh.net) contributors: [@benhylau](https://github.com/benhylau), [@darkdrgn2k](https://github.com/darkdrgn2k), [@dcwalk](https://github.com/dcwalk), [@uditvira](https://github.com/uditvira), [@Shrinks99](https://github.com/Shrinks99), and [@Pedro-on-a-bike](https://github.com/Pedro-on-a-bike). While most of the material is created new, the course also incorporates many prior works listed [here]({{ site.baseurl }}/articles/general/general-extra-resources.pdf), or otherwise referenced in their individual course modules.
```
Comment (dcwalk) [inline README.md]: baseurl issue

**Comment by dcwalk** [comment]
Also @benhylau -- totally fine to remove the course design stuff from readme, but I think it should be kept in the repo. In part to revisit/reflect on in future design iterations. 

Maybe a `course-design` folder? 
Or a `planning` folder and could move some stuff from `general` there? Also, general is kind of a weird name

**Comment by benhylau** [comment]
Yea I had to choose between breaking README rendering on GitHub or hardcoding a path, but I suppose the latter is not that bad since the file originate from outside the `site` folder, and your point about people arriving to GitHub is very true. I will change to hardcode.

The only course design lines removed were:

>- Create syllabus with sufficient flexibility so it can be adapted to Toronto Libraries in other areas and other settings
>- With a community network project as basis, establish regular hack nights at the Toronto Libraries' Innovation Hubs
>- Engage students and equip them with necessary knowledge to continue involvement in local mesh community efforts
>- Publish this as open-source syllabus that other communities can adopt to kick-start a mesh community

I think the first two are specific to a particular facilitation and no longer fit in the repo. If those are Toronto goals they should be tracked in some facilitation planning board like other facilitation efforts?

Perhaps we can integrate latter two points somewhere on README?

**Comment by benhylau** [comment]
Addressed comments and added new issue to add back course design lines. Merging since this contains significant changes.

**Pull request merged**